docs/docs/add-a-service-worker.md

```diff
@@ -4,7 +4,7 @@ title: Add a Service Worker
 
 ### What is a service worker
 
-A service worker is a script that your browser runs in the background, separate from a web page, opening the door to features that don't need a web page or user interaction.They increase your site availability in spotty connections, and are essential to making a nice user experience.
+A service worker is a script that your browser runs in the background, separate from a web page, opening the door to features that don't need a web page or user interaction. They increase your site availability in spotty connections and are essential to making a nice user experience.
```

Comment:
This doc will be removed and its contents will be merged with another doc. You can track the progress (#9263) there.

I do like the changes you made, so I think I will incorporate them into my PR, if I can!

Thank you so much for this PR! Please sift through other docs and see if you find anything else to improve! 💖

Reply:
nice catch!